builder/builder.js

```diff
@@ -37,41 +38,48 @@ module.exports = (options) => {
     cssDir,
     environment = 'development',
     jsDir,
-    output = defaultOutputThemeFile,
-    themeFile,
     watch,
   } = options;
 
-  const themeFiles = [defaultThemeYamlPath].concat(themeFile).filter(Boolean);
+  const bundles = [{
+    themeFiles: [defaultThemeYamlPath],
+    output: path.join(defaultOutputThemeDir, 'default'),
+  }];
+  getAvailableSets().forEach((themeOptions) => {
+    bundles.push({
+      themeFiles: [defaultThemeYamlPath].concat(getThemeFiles(themeOptions)),
+      output: path.join(defaultOutputThemeDir, `${themeOptions.brand}-${themeOptions.affiliate}`),
```

Comment:
this will result with breaking change .. just keep in mind that in `rwd` we have usage of the theme.css